### PR TITLE
[CP-1412] I want to be able to see the status of my message

### PR DESCRIPTION
**Pull request by wercix**
Jira: [CP-1412]

**Description**
Add sending status
<details>
<summary><b>Screenshots</b></summary>

Screen Shot 2022-09-19 at 15 29 47

</details>

[CP-1412]: https://appnroll.atlassian.net/browse/CP-1412?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dkarski — DISMISSED** [review]

File: packages/app/src/messages/presenters/message.presenter.ts [hunk 1]
```diff
@@ -63,15 +63,14 @@ export class MessagePresenter {
   private static getMessageType(
     messageType: AcceptablePureMessageType
   ): MessageType {
-    if (
-      messageType === PureMessageType.QUEUED ||
-      messageType === PureMessageType.OUTBOX
-    ) {
+    if (messageType === PureMessageType.OUTBOX) {
```
Comment (igorbezsmertnyi) [inline packages/app/src/messages/presenters/message.presenter.ts:66]: Please add a test for the `presenter`

File: packages/app/src/messages/presenters/message.presenter.ts [hunk 2]
```diff
@@ -63,15 +63,14 @@ export class MessagePresenter {
   private static getMessageType(
```
Comment (igorbezsmertnyi) [inline packages/app/src/messages/presenters/message.presenter.ts:63]: And also, please made the same changes in `packages/app/src/data-sync/presenters/message/message.presenter.ts`

**Review by igorbezsmertnyi — APPROVED** [review]

**Review by igorbezsmertnyi — CHANGES_REQUESTED** [review]
👆

**Review by wojcik92michal — COMMENTED** [review]
Nothing from my side, awaiting for changes suggested by Igor

File: packages/app/src/messages/presenters/message.presenter.ts (hunk 1 above)
Comment (wercix) [reply packages/app/src/messages/presenters/message.presenter.ts:66]: Test in [005e09b](https://github.com/mudita/mudita-center/pull/981/commits/005e09bffe30858f82fce48686253fc8915cc4ca)

File: packages/app/src/messages/presenters/message.presenter.ts (hunk 2 above)
Comment (wercix) [reply packages/app/src/messages/presenters/message.presenter.ts:63]: Added in [005e09b](https://github.com/mudita/mudita-center/pull/981/commits/005e09bffe30858f82fce48686253fc8915cc4ca), thanks!

**Review by dkarski — APPROVED** [review]

**Review by wojcik92michal — APPROVED** [review]

**Review by igorbezsmertnyi — APPROVED** [review]

**Pull request merged**